src/platform/silabs/rs911x/rsi_ble_config.h

```diff
@@ -14,9 +14,7 @@
  * sections of the MSLA applicable to Source Code.
  *
  ******************************************************************************/
-
-#ifndef RSI_BLE_CONFIG_H
-#define RSI_BLE_CONFIG_H
+#pragma once

```

Comment:
Removing the guards breaks compatibility with plain C code. I suggest to rename the guards instead:
```
#ifndef RSI_BLE_CONFIG_RS911X_H
#define RSI_BLE_CONFIG_RS911X_H
```

Reply:
that doesn't solve this issue as stated in the PR description

Reply:
Also pragma is compatible with plain C code, it's simply a matter of compilater :)